src/server/api/lib/twilio.js

```diff
@@ -131,8 +134,30 @@ async function sendMessage(message) {
       statusCallback: process.env.TWILIO_STATUS_CALLBACK_URL
     }, parseMessageText(message))
 
-    if (process.env.TWILIO_MESSAGE_VALIDITY_PERIOD) {
-      messageParams.validityPeriod = process.env.TWILIO_MESSAGE_VALIDITY_PERIOD
+    let twilioValidityPeriod = process.env.TWILIO_MESSAGE_VALIDITY_PERIOD
+
+    if (message.send_before) {
+      // the message is valid no longer than the time between now and
+      // the send_before time, less 30 seconds
+      // we subtract the MESSAGE_VALIDITY_PADDING_SECONDS seconds to allow time for the message to be sent by
+      // a downstream service
+      const messageValidityPeriod = Math.ceil((message.send_before - Date.now())/1000) - MESSAGE_VALIDITY_PADDING_SECONDS
```

Comment:
it looks like `message.send_before` in [schema.js::sendMessage](https://github.com/MoveOnOrg/Spoke/pull/1025/files#diff-e3e0c0fe13e813c7748359d6a931a6faR944) elsewhere in this diff is set to UTC.  So does `Date.now()` need to be in UTC?

Reply:
👍 

#1029 created to track this.  I humbly suggest that this not delay merging this PR, because the servers are configured with local time == UTC.